test/input/markdown.md

```diff
@@ -1,4 +1,5 @@
 # Header 1
+I'm u's'i'n'g a'p'o's't'r'o'p'h'e's
```

Comment:
Make sure to include an example where it will actually apply each of the different smart quote types, for example:

```md
- 'single quotes' work
- "double quotes" work
- I'm an apostrophe
```